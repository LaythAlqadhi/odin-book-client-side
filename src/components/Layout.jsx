import React from 'react';

const Layout = ({ children }) => {
  return (
    <div>
      <header>
        <h1>Odinbook</h1>
      </header>
      <main>{children}</main>
      <footer>
        <p>Odinbook 2024</p>
      </footer>
    </div>
  );
};

export default Layout;
