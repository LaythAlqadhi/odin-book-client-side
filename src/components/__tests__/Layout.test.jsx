import React from 'react';
import { render, screen } from '@testing-library/react';
import Layout from '../Layout';

describe('Layout component', () => {
  it('renders Layout component correctly', () => {
    render(<Layout/>);
    
    expect(screen.container).toMatchSnapshot();
  });
});
