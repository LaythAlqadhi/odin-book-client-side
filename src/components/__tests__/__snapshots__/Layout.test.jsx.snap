// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Layout component > renders Layout component correctly 1`] = `undefined`;
